Makefile

```diff
@@ -93,27 +111,32 @@ binary: $(CALICOCTL_FILES) vendor
 	fi; \
 	GOOS=$(OS) GOARCH=$(ARCH) CGO_ENABLED=0 go build -v $$INSTALL_FLAG -o dist/calicoctl-$(OS)-$(ARCH) $(LDFLAGS) "./calicoctl/calicoctl.go"
 
-dist/calicoctl: $(CALICOCTL_FILES) vendor
+
+dist/calicoctl:
```

Comment:
These have lost some dependencies so they won't get rebuilt if the code changes.  Please can you fix up?

Reply:
But if `dist/calicoctl` already exists and its target has no dependencies then make will think it is up to date, when it's not.

Reply:
(oops, hit wrong `Enter` command)

> which is why our releases include some asymmetrically named binaries.

I can see that. It _is_ a bit annoying.

> I'd be more than happy to remove this target and call it what it is - calicoctl-linux-amd64

For which we already have a target, which makes it even easier. 

> perhaps we start by leaving this and attaching both calicoctl and calicoctl-linux-amd64 to the next release

Ah, so we have both in the docs and the actual releases, and then can remove the "arch-less" one at our leisure, rather than holding up? That makes sense. I can update the makefile part too.

So I think our path here is:

1. Update the `make help` so it tells us to do the right thing and copy all of the correct files
2. Replace the `cp` to do so _only_ if it is `calicoctl-linux-amd64`
3. Add the dependencies back.

That it?

Reply:
Even better. `calicoctl` is _only_ for `linux-amd64`, so let's simplify that target.

Reply:
Updated

Reply:
Not sure I see the value in arch named tarball with single file inside vs arch named file. Either way, can we do it as separate PR?